modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -4337,8 +4398,6 @@ definition {
 		}
 	}
 
-
-
```

Comment:
Instead of fixing your SF error in the next commit, this should be fixed in the original commit.